docker/Dockerfile

```diff
@@ -45,6 +45,7 @@ EXPOSE 8008 8010
 # ENTRYPOINT to be the the `functions start` command.  Deployed functions persist across emulator restarts.
 RUN functions start \
     && functions deploy martha_v2 --trigger-http \
+    && functions deploy martha_v3 --trigger-http \
```

Comment:
There should probably be an update to the README too, if just a placeholder that says "martha_v3: In early development, don't use"?

Reply:
Oops, I forgot to update README. Will do that.

Reply:
done!